.github/workflows/ci.yml

```diff
@@ -55,6 +55,11 @@ jobs:
       - name: Run actionlint
         shell: bash
         run: scripts/actionlint.sh
+      - name: Ensure consistent avalanchego version
+        shell: bash
+        run: |
+          ./scripts/update_avalanchego_version.sh
```

Comment:
should we move this to e2e step? and set `CUSTOM_ACTION` there?

Reply:
Sorry I got this completely wrong. Thought this was being used only in that step.